bot/exts/utils/snekbox/_cog.py

```diff
@@ -373,11 +382,39 @@ async def send_job(self, ctx: Context, job: EvalJob) -> Message:
 
                 msg += f"\n{Emojis.failed_file} {blocked_msg}"
 
+            # Split text files
+            text_files = [f for f in allowed if f.suffix in TXT_LIKE_FILES]
+            # Inline until budget, then upload to paste service
+            budget_lines = MAX_OUTPUT_BLOCK_LINES
+            budget_chars = MAX_OUTPUT_BLOCK_CHARS
+            for file in text_files:
+                file_text = file.content.decode("utf-8", errors="replace") or "[Empty]"
```

Comment:
Not a big deal, but I think it would be nicer if `format_output` would take a placeholder string if the output is empty, instead of the `mark_no_output` flag.

Reply:
Refactored to `output_default` str in [dc7653b](https://github.com/python-discord/bot/pull/2326/commits/dc7653b63deee4abcf850b28fc4d30edf04b5e27)